industry_benchmarks/utils/maint/check_protein_prep.py

```diff
@@ -0,0 +1,77 @@
+import numpy as np
+import pathlib
+import click
+import MDAnalysis as mda
+from MDAnalysis.lib.distances import distance_array
+
+
+@click.command
+@click.option(
+    '--prepared',
+    type=click.Path(dir_okay=False, file_okay=True, path_type=pathlib.Path),
+    required=True,
+    help="Path to the prepared PDB file",
+)
+@click.option(
+    '--original',
+    type=click.Path(dir_okay=False, file_okay=True, path_type=pathlib.Path),
+    required=True,
+    help="Path to the original PDB file",
+)
+def run(prepared, original):
+    """
+    Compare the prepared PDB with its original form, checking that there
+    are not major differences in atom counts and positions.
+
+    Parameters
+    ----------
+    prepared : pathlib.Path
+      A Path to the prepared PDB file.
+    original : pathlib.Path
+      A Path to the `original` PDB file.
+    """
+    prep = mda.Universe(prepared)
+    prev = mda.Universe(original)
+
+    prep_prot = prep.select_atoms('protein and not resname ACE NME NMA')
+    prev_prot = prev.select_atoms('protein and not resname ACE NME NMA')
+
+    for resi, resj in zip(prep_prot.residues, prev_prot.residues):
+        if len(resi.atoms) != len(resj.atoms):
```

Comment:
The number of atoms for the residues next to the caps (so first and last aa) will change when adding caps. Would it make sense to filter those out?